README.md

```diff
@@ -1,2 +1,16 @@
-# Flutter-Minimal-Integration
-Minimal Integration App demoing how to integrate PointSDK Flutter Plugin
+# flutter_minimal_app
```

Comment:
Should be updated to our README, not a template

Reply:
I have updated this